Core/Core/Common/CommonUI/ViewModifiers/FirstAppearViewModifier.swift

```diff
@@ -18,8 +18,8 @@
 
 import SwiftUI
 
-extension View {
-    public func onFirstAppear(perform action: (() -> Void)? = nil) -> some View {
+public extension View {
```

Comment:
This is also against the [guidelines](https://github.com/instructure/canvas-ios/blob/master/.github/guidelines/swift-style-guide.md#extension-access-levels).